src/Microsoft.Extensions.Localization/IApplicationNameProvider.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Localization
+{
+    public interface IApplicationNameProvider
```

Comment:
Hmm, I do wonder - which of the Entropy (or other?) samples need to be updated due to this? I admit I hadn't quite considered those implications.